docs/getting_started.Rmd

```diff
@@ -91,14 +91,14 @@ If you find yourself routinely pinning data larger that this, you might need to
 
 ```{note}
```

Comment:
Shoot -- I just did some testing, and am realizing that because we're using Rmd, we need to escape this, like:

``````
<!-- #region -->
```{note}
abc
```
<!-- #endregion -->
``````

jupyterbook uses its own custom markdown format, or notebooks, so we are using jupytext to convert Rmd to notebooks for the docs. jupytext thinks this note block is source code where `note` is the engine.

https://jupyterbook.org/en/stable/file-types/index.html

Reply:
happy to wrap up the cleanup and merge if useful!

Reply:
fixed!